Test/Unit/Visitor/Arithmetic.php

```diff
@@ -101,4 +99,89 @@ public function case_visitor_exhaustively()
                 }
             });
     }
+
+    public function case_visitor_variable()
```

Comment:
I wonder if we should have 2 test cases here: One for an unknown variable, one with a declared variable.